charts/cluster-autoscaler/README.md

```diff
@@ -309,6 +309,25 @@ Containers:
 
 Though enough for the majority of installations, the default PodSecurityPolicy _could_ be too restrictive depending on the specifics of your release. Please make sure to check that the template fits with any customizations made or disable it by setting `rbac.pspEnabled` to `false`.
 
+### VerticalPodAutoscaler
+
+The chart can install a VPA for the Deployment if needed. A VPA can help minimize wasted resources when usage spikes periodically.
+The following example snippet can be used to install a `autoscaling.k8s.io/v1` VPA.
```

Comment:
Can we link out to the VPA docs please, there's a reasonable chance users will be reading this Readme disconnected from this repo, and may want to find out more about the VPA.